web/redux/actions/chat-actions.js

```diff
@@ -86,3 +87,14 @@ export function loadedMoreMessages(room, messages) {
 export function failedLoadingMoreMessages(error) {
   return { type: actions.CHAT_FAILED_LOADING_MESSAGES, error };
 }
+
+export function handleDeleteMessage(uuid) {
+  return (dispatch) => {
+    API.deleteMessage(uuid).then(() => {
+      console.log('Sent delete message request.');
```

Comment:
Should probably remove the console statements. What I've done in the past is dispatch actions with no reducer handling (which would make them visible in the redux devtools but not to the end user).